packages/subgraph/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "lib": ["DOM","ES2018"],
+    "types": ["@graphprotocol/graph-ts", "node"],
+    "skipLibCheck": true,                     /* Skip type checking of declaration files. */
+    "forceConsistentCasingInFileNames": true,  /* Disallow inconsistently-cased references to the same file. */
+    "strictNullChecks": true
+  },
+  "exclude": [
+    "src/types"
+  ]
```

Comment:
❓

Reply:
It's the auto-generated types folder... Should we preserve it on the compilation?